enforcer-rules/src/it/fail-build-for-linkage-errors/verify.groovy

```diff
@@ -20,7 +20,7 @@ assert buildLog.contains('''\
 ''')
 
 assert buildLog.contains('''\
-[ERROR] Problematic artifacts in the dependency tree:
+Problematic artifacts in the dependency tree:
```

Comment:
Because linkage problems and the dependency paths are printed as one message, this no longer carry `[ERROR] `.